frontend/src/components/auth/Login.tsx

```diff
@@ -78,7 +78,11 @@ const Login = (): React.ReactElement => {
   };
 
   if (authenticatedUser) {
-    return <Redirect to={HOME_PAGE} />;
+    return authenticatedUser.role === "Teacher" ? (
```

Comment:
i'm seeing some funky behaviour on my end bc my google email is associated with a role from my previous project that doesn't exist in Jump Math lol (called "donor") and now `authenticatedUser.role == donor` for me. its a very edge case but overall might be better to use a switch statement here specifying only "Teacher" and "Admin" cases + default
```suggestion
        switch (authenticatedUser.role) {
      case "Teacher":
        return <Redirect to={TEACHER_PAGE} />;
      case "Admin":
        return <Redirect to={ADMIN_PAGE} />;
      default:
        return <Redirect to={COMPONENT_LIBRARY} />;
    }
```

Reply:
wait LOL, how were you able to sign up as a donor? did you use Google auth? 🧐

Reply:
yea my google account - it just auto authenticated me lol (i'm using chrome)

Reply:
okay - let's leave this as is for now. I'll look into Google Auth behaviour and create any subsequent tickets to resolve this error! cc @j985chen